sources/engine/Stride.Graphics/OpenGL/Buffer.OpenGL.cs

```diff
@@ -168,7 +168,7 @@ protected void Init(IntPtr dataPointer)
                     if ((Flags & BufferFlags.ShaderResource) != 0)
                     {
 #if STRIDE_GRAPHICS_API_OPENGLES
-                        Internal.Refactor.ThrowNotImplementedException();
+                        throw new NotImplementedException();
```

Comment:
I meant to use NotSupportedException in all similar cases. I didn't comment every single file.

Reply:
Right, I updated branch now. Let me know if that's okay 😅